### PR TITLE
Content Views Trend

### PR DESCRIPTION
**Pull request by MykolaGalian**
[US123103](https://rally1.rallydev.com/#/23525809118d/iterationstatus?detail=%2Fuserstory%2F462189600204&sharedViewId=328666453148&fdp=true?fdp=true): [Engagement][UserDrill] Content Views Trend

With zoom

First PR with test data.
Kevin agreed to use the current date instead of the word "Today". So will need to add the current date to data (if it does not exist with 0 views).

 With a flag that hides this stuff for now (on for Demo and tests only).

Functional:
Local App work as expected with test data.
Browsers (Windows 10 Chrome, Firefox, Edge)

Accessibility: n/a
Security: n/a
Devops: n/a
UI: Kevin approved

FYI @johngwilkinson @anhill-D2L @bemailloux

### CODE REVIEW

File: components/content-views-card.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	get _dateText() {
+		return this.localize('contentViewsCard:date');
+	}
+
+	render() {
+		// NB: relying on mobx rather than lit-element properties to handle update detection: it will trigger a redraw for
+		// any change to a relevant observed property of the Data object
+		return html`
+			<div class="d2l-insights-content-views-title d2l-skeletize d2l-skeletize-45 d2l-body-standard">${this._cardTitle}</div>
+			<d2l-labs-chart class="d2l-insights-summary-card-body" .options="${this.chartOptions}" ?skeleton="${this.skeleton}"></d2l-labs-chart>`;
```
Comment (Vitalii-Misechko) [inline components/content-views-card.js:74]: `d2l-insights-summary-card-body` is not defined

**Comment by johngwilkinson** [comment]
> Visual diff tests failed - pull request #303 has been opened with the updated goldens.

FYI, please don't merge these PRs unless the change is expected: https://github.com/Brightspace/insights-engagement-dashboard/pull/297#issuecomment-754096806

File: components/content-views-card.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+				enabled: false
+			},
+			xAxis: {
+				tickInterval:  7 * 24 * 3600 * 1000, //week
+				type: 'datetime',
+				labels: {
+					format: '{value:%b %e/%y}',
+					rotation: -60,
+					y: 30,
+					align: 'center'
```
Comment (Vitalii-Misechko) [inline components/content-views-card.js:106]: I'd suggest removing `y` and `align` option. It looks better IMO. Have a look at my PR screenshot #304. 
Anyway should be approved by Kevin

File: components/user-drill-view.js [hunk 3]
```diff
@@ -364,6 +365,7 @@ class UserDrill extends SkeletonMixin(Localizer(MobxLitElement)) {
 
 				.cards="${this.summaryCards}"
 			></d2l-summary-cards-container>
+			<div><d2l-insights-content-views-card .data="${this._data}" ?skeleton="${this._isLoading}"></d2l-insights-content-views-card></div>
```
Comment (Vitalii-Misechko) [inline components/user-drill-view.js:368]: I guess you don't need additional `div`.

**Review by Vitalii-Misechko — APPROVED** [review]
LGTM

**Comment by Vitalii-Misechko** [comment]
I'm not sure if we should merge UI part into the master. It seems we're going to release these stories in the next release.

**Comment by johngwilkinson** [comment]
> I'm not sure if we should merge UI part into the master. It seems we're going to release these stories in the next release.

Recap from standup: it would be good to merge so the code doesn't go stale, but with a flag on the dashboard that hides this stuff for now (on for tests only).

**Comment by NicholasHallman** [comment]
@MykolaGalian can you add the isDemo flag to this PR? Then I'll approve

**Review by NicholasHallman — APPROVED** [review]

**Review by Vitalii-Misechko — APPROVED** [review]

**Pull request merged**